src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
+import { Comment } from '../types/Comment';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<Props> = ({comments}) => {
+  return (
+  <>
+  {comments.map((comment) => <CommentInfo comment={comment} key={comment.id}/>)}
```

Comment:
You can write like this
```suggestion
 {comments.map(comment => (
      <CommentInfo comment={comment} key={comment.id} />
    ))}
```